serde-proto/protobuf/v1/common_types.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+package v1.common_types;
+
+
+message Timestamp {
```

Comment:
We don't need to define our own, we just add
import "google/protobuf/timestamp.proto";
and then use this `Timestamp` ?

Reply:
Yes. We need to also add
extern_path(".google.protobuf.Timestamp", "::serde_proto::common_types::Timestamp")
 while build the protobuf to use this timestamp